src/PushNotifications.Contracts/PushNotifications/Delivery/IPushNotificationBufferedDelivery.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace PushNotifications.Contracts.PushNotifications.Delivery
+{
+    public interface IPushNotificationBufferedDelivery
```

Comment:
why is there "buffered" in the name? isn't this an implementation detail?

Reply:
Implementations using this interface are capable of sending multiple PN at once. We buffer/stack this messages before flushing them. e.g InMemoryBufferedDelivery